tests/app/oauth/test_rest.py

```diff
@@ -664,3 +664,10 @@ def test_returns_401_on_any_exception(self, client, mock_va_sso_authorize_access
         assert response.status_code == 401
         assert response.json == {"error": "Unauthorized", "description": "Authentication failure"}
         assert mock_statsd.incr.called_with('oauth.authorization.failure')
+
+
+class TestGetServicesByUser:
+    def test_should_return_200(self, client, sample_user):
+        json_resp = client.get('/auth/my-services/{}'.format(str(sample_user.id)))
```

Comment:
`str` is called on anything passed to `format`, so you don't explicitly need to call it.